### PR TITLE
updated vendors

### LINKED ISSUE #124: Consider relaxing phpunit version requirement?
I'm unable to add this project as it requires `phpunit` ~3.7, while other projects I'm using have a 4.* requirement.  Could that be updated to a >=3.7 or is there a reason why the tests are tied to this version?

### PR DESCRIPTION
**Pull request by lsv**
* Updated all vendors
* Updated phpunit to version ~4

Fixes #124

### CODE REVIEW

**Comment by stof** [comment]
@justinrainbow should we remove the lock file from the repo to make sure we can install things on PHP 5.3.3 as well ? the current lock file requires 5.3.23+

**Comment by bighappyface** [comment]
+1, we don't need to store the `composer.lock` file and can instead specify the versions of packages in `composer.json` itself. Pull it out and lets see how the build goes.

**Comment by keradus** [comment]
ref #139

**Comment by bighappyface** [comment]
@lsv please rebase per #139

File: composer.json [hunk 1]
```diff
@@ -40,7 +40,7 @@
     },
     "require-dev": {
         "json-schema/JSON-Schema-Test-Suite": "1.1.0",
-        "phpunit/phpunit": "~3.7",
+        "phpunit/phpunit": "~4",
```
Comment (GrahamCampbell) [inline composer.json]: Should really be `~4.0`

File: composer.json [hunk 2]
```diff
@@ -40,7 +40,7 @@
...
         "phpdocumentor/phpdocumentor": "~2"
```
Comment (GrahamCampbell) [inline composer.json]: ~2.0

**Comment by bighappyface** [comment]
Closed with https://github.com/justinrainbow/json-schema/pull/174

**Pull request closed without merging**